src/goto-symex/builtin_functions.cpp

```diff
@@ -1507,3 +1507,97 @@ void goto_symext::intrinsic_get_object_size(
     false,
     cur_state->guard);
 }
+
+void goto_symext::bump_call(
```

Comment:
It would be great if this function could be used instead of the lambdas in goto_symext::intrinsic_memset() as well.